tests/integration/targets/inventory/playbook.yml

```
@@ -0,0 +1,19 @@
+- hosts: localhost
+  tasks:
+    - name: "test option: --list --export"
+      shell: ansible-inventory --list --export --inventory inventory.sops.yml
```

```suggestion
      command: ansible-inventory --list --export --inventory inventory.sops.yml
```
Never use `shell` when `command` suffices.